lib/src/robot/client.dart

```diff
@@ -1,36 +1,35 @@
 import 'package:grpc/grpc_connection_interface.dart';
-import 'package:viam_sdk/src/gen/robot/v1/robot.pbgrpc.dart';
-import 'package:viam_sdk/src/resource/manager.dart';
-import 'package:viam_sdk/viam_sdk.dart';
 
+import '../domain/web_rtc/web_rtc_client/web_rtc_client.dart';
+import '../gen/common/v1/common.pb.dart';
+import '../gen/robot/v1/robot.pbgrpc.dart';
 import '../media/stream/client.dart';
+import '../resource/base.dart';
+import '../resource/manager.dart';
+import '../resource/registry.dart';
+import '../rpc/dial.dart';
+import '../viam_sdk.dart';
```

Comment:
I think we can add a lint rule to prefer relative imports. 👀  but maybe we don't want that because we'll want to do package imports in our example code. something to think about!